### PR TITLE
added vOLE example & fixed bug in ring/uniform sampler

### PR DESCRIPTION
**Pull request by Pro7ech**
Todo : 

- [x] Document
- [x] Update makefile

### CODE REVIEW

File: examples/ring/vOLE/main.go [hunk 1]
```diff
@@ -0,0 +1,406 @@
+package main
```
Comment (ChristianMct) [inline examples/ring/vOLE/main.go:1]: There needs to be an extensive godoc that describes this example and provides links to the paper for further reference.

File: examples/ring/vOLE/main.go [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+	for i := range lns.coeffs {
+		lns.coeffs[i] = ring.RandInt(bound)
+	}
+
+	lns.baseRing.SetCoefficientsBigint(lns.coeffs, pol)
+
+	return
+}
+
+func main() {
```
Comment (ChristianMct) [inline examples/ring/vOLE/main.go:100]: The example needs to be added to the `Makefile`'s `test_example` target.

**Review by ChristianMct — CHANGES_REQUESTED** [review]

File: examples/ring/vOLE/main.go (hunk 2 above)
Comment (Pro7ech) [reply examples/ring/vOLE/main.go:100]: Right!

**Review by jrtroncoso — APPROVED** [review]
I've made a few changes in the vOLE example file. The rest of the updates look good

**Review by ChristianMct — APPROVED** [review]

**Pull request merged**